initfiles/componentfiles/configschema/xsd/thor.xsd

```diff
@@ -94,8 +94,8 @@
                                           hpcc:tooltip="Verify that a read file's CRC matches the published meta data CRC"/>
                             <xs:attribute name="crcWriteEnabled" type="xs:boolean" hpcc:displayName="CRC Write Enabled" hpcc:presetValue="true"
                                           hpcc:tooltip="Calculate and publish a CRC per published disk output file"/>
-                            <xs:attribute name="slaveDaliClient" type="xs:boolean" hpcc:displayName="Slave Dali Client" hpcc:presetValue="false"
-                                          hpcc:tooltip="Set to true if thor slaves require dali connectivity"/>
+                            <xs:attribute name="allowDaliAccess" type="xs:boolean" hpcc:displayName="Worker Dali Client" hpcc:presetValue="false"
```

Comment:
Should there be similar entries for the new sasha options?

Reply:
it's "documentation" only, all Debug options will be copied regardless by thor.xsl. But guess should add (even though don't want to encourage).